### PR TITLE
Feature/iac 604 tls support

### PR DESCRIPTION
**Pull request by mmidolesov**
### Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added relevant error handling and logging messages to help troubleshooting
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced
- [x] I have tested against live environment, if applicable
- [x] I have added relevant usage information (As-built)
- [ ] Any structure and/or content vRA-NG improvements are synchronized with vra-ng and ts-vra-ng archetypes
- [ ] Every new or updated Installer property is documented in docs/archive/doc/markdown/use-bundle-installer.md
- [ ] Dependencies in pom.xml are up-to-date
- [ ] My changes have been rebased and squashed to the minimal number of relevant commits
- [ ] My changes have a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### CODE REVIEW

File: typescript/polyglotpkg/src/lib/model.ts [hunk 1]
```diff
@@ -53,7 +53,8 @@ export type PlatformDefinition = PackageDefinition & {
         base?: string,
         tags?: Array<string>
         memoryLimitMb?: number,
-        timeoutSec?: number
+        timeoutSec?: number,
+        protocolType?: 'Ssl3' | 'Tls' | 'Tls11' | 'Tls12' | 'Tls13'
```
Comment (Michaelpalacce) [inline typescript/polyglotpkg/src/lib/model.ts:57]: How can we set this in a polyglot project?

File: docs/versions/latest/Release.md [hunk 2]
```diff
@@ -23,12 +23,25 @@
 [//]: # (Describe the feature)
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
+### *Enforce TLS 1.2 when Downloading PowerShell Modules*
+Added support for downloading modules in powershell polyglot project handlers through 'Ssl3' | 'Tls' | 'Tls11' | 'Tls12' | 'Tls13'
 
-
+#### Previous Behavior
```
Comment (Michaelpalacce) [inline docs/versions/latest/Release.md:29]: Notes like:

- IT didn't work
- Now it works

Don't really provide good information.

Try to explain how it was fixed and why it was failing before

File: typescript/polyglotpkg/src/lib/model.ts (hunk 1 above)
Comment (Michaelpalacce) [reply typescript/polyglotpkg/src/lib/model.ts:57]: Perhaps it's good to document this as well?

File: typescript/polyglotpkg/src/strategies/powershell.ts [hunk 3]
```diff
@@ -93,17 +93,22 @@
...
             }
         });
 
         this.logger.info(`Powershell modules included: ${modules}`);
 
+        const moduleNames = modules.join(",");
+        const securityProtocolArg = polyglotJson.platform.protocolType ? `[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::${polyglotJson.platform.protocolType};` : "";
+        const command = `${securityProtocolArg} Save-Module -Name ${moduleNames} -Path '${modulesPath}' -Repository PSGallery`;
+        
+        const args = polyglotJson.platform.protocolType ? [`-c "${command}"`] : [command];
```
[the remark this answers is not in the document]
Comment (Michaelpalacce) [reply typescript/polyglotpkg/src/strategies/powershell.ts:107]: I don't think this is an actual issue in our case?

**Review by Michaelpalacce — DISMISSED** [review]
A few things left

**Review by n-vodenicharski — APPROVED** [review]
all good

**Review by n-vodenicharski — APPROVED** [review]

File: typescript/polyglotpkg/src/lib/model.ts (hunk 1 above)
Comment (mmidolesov) [reply typescript/polyglotpkg/src/lib/model.ts:57]: Done

**Pull request merged**